src/utils/getPrepareTodos.ts

```diff
@@ -0,0 +1,27 @@
+import { SortFields } from '../types/SortFields';
+import { Todo } from '../types/Todo';
+
+export const getPreparedTodos = (
+  todoList: Todo[],
+  select: SortFields,
+  query: string,
+) => {
+  const visibleTodos = [...todoList];
+
+  return visibleTodos
+    .filter(todo =>
+      todo.title.toLowerCase().includes(query.trim().toLowerCase()),
+    )
+    .filter(({ completed }) => {
```

Comment:
- If you don't have any query you don't need to filter
- The same if the filter type is `all`